docs/src/components/LandingNavigation.tsx

```diff
@@ -53,13 +51,11 @@ const LogoContainer = styled.a`
   box-sizing: border-box;
   background-color: ${tm(({ colors }) => colors.neutral0)};
   border: none;
-  &:hover {
-    cursor: pointer;
-  }
+  cursor: pointer;
 `;
 
 const LandingNavigation: FC = () => {
-  const [isMobileMenuOpen, setMobileMenuState] = useState(false);
+  const [isMobileMenuOpen, setIsMobileMenuOpen] = useState(false);
```

Comment:
for better code readablity it's recommended to keep same names for a state variable and a function